addon/templates/components/frost-table-header.hbs

```diff
@@ -1,10 +1,14 @@
 {{! Template for the frost-table-header component }}
+{{! frost-fixed-table's columns are broken in to several separate rows,
+    so if we encounter that we want to use it instead of index. }}
 {{#each columns as |column index|}}
   {{frost-table-cell
+    callbackDispatch=(action 'handleCallback' (or column.colIndex index))
```

Comment:
If I know that `column.colIndex` will always exist, that is certainly simpler.  However, `frost-table` doesn't yet add those columns, and so I was treating the indexes as only present when they are used by `frost-fixed-table`.  Because of the simplifications, it's probably better to add the feature to the normal frost table as well, and then make it required as you suggested.